pkg/piecestore/psserver/maxreader.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package psserver
+
+import (
+	"errors"
+	"io"
+)
+
+// MaxReader behaves like a normal reader, but throws an error if the data
```

Comment:
State the exact error class that will be returned by Read. See my other comment for the actual error.